azext_iot/common/utility.py

```diff
@@ -280,3 +294,14 @@ def calculate_millisec_since_unix_epoch_utc():
     now = datetime.utcnow()
     epoch = datetime.utcfromtimestamp(0)
     return int(1000 * (now - epoch).total_seconds())
+
+
+def get_sas_token(target):
```

Comment:
This function is so specific to Digital Twin auth. It makes sense to add a class method to `DigitalTwinSasTokenAuthentication` so you can do something like `DigitalTwinSasTokenAuthentication.from_target(target)` where `from_target` is a class method.

Reply:
Also...don't need to do it in this PR, but we should think about shortening the names of the auth elements...they are quite verbose especially when the module name h as a lot of the same info as the class name.

Reply:
correct, I'll issue a new PR against it.